copter/source/docs/traditional-helicopter-connecting-apm.rst

```diff
@@ -1,204 +1,98 @@
-don't recommend using an external gyro and in time support for it will
-be removed.
-
-Advanced Configuration of your Receiver and Servos
-==================================================
-
-Copter can be configured to function properly with your radio set up
-with a helicopter profile. The reason for using this setup is to gain
-the benefit of helicopter functions pre-programmed into your radio, such
-as Idle-Ups and Throttle Hold. It is mandatory to set your swash plate
-to a non-CCPM mixing type. This is because the Copter program performs
-the CCPM mixing internally, and it must see discreet radio commands for
-Aileron, Elevator, Rudder and Collective Pitch.
-
-The full details of this setup is outside the scope of this Wiki page.
-Chances are if you have been flying RC helicopters for a while, you know
-exactly what you need to do already.
-
-With a Futaba radio this would be an H-1 or SW1 depending on your radio
-model. Other brands of radio will use a different nomenclature such as
-NOR, NORM, or 1SERVO. It is critical that CCPM mixing is turned off,
-please consult your radio manual to ensure you have it set up correctly.
-
-**Please note** that many of the helicopter mixing functions in your
-radio (such as Revo Mixing) need to be turned off. This is why this
-setup is only recommended for advanced users.
+.. image:: ../images/PH21_3.jpg
+    :target: ../_images/PH21_3.jpg
+
+The Pixhawk2.1 is the latest controller that runs the ArduPilot flight stack. It
+is highly recommended for helicopters. It has much better internal damping of
+the IMU's for vibration, triple redundant heated IMU's, redundant barometers and
+compasses. Experience has shown the tuning, handling and stability performance
+of your helicopter will be greatly improved over the first generation Pixhawk.
+
+Before you begin connecting the system it is recommended to review the docs for
+the Pixhawk 2.1 available `here  <http://www.proficnc.com/index.php?controller=attachment&id_attachment=8>`_
+
+Overview of servo, and RX connection
+====================================
+
+The RC input to the Pixhawk is either PPM SUM, which will give you 8 channels.
+Or S.Bus, which will give you up to 18 channels. If you have a receiver that
+outputs only PWM, you will need a PPM encoder to connect it to Pixhawk.
+
+The default RC input and SERVO output mapping is as follows:
+
++--------------+-------------+
+| RC Channel   | SERVO output|
++--------------+-------------+
+| 1            | Aileron     |
++--------------+-------------+
+| 2            | Elevator    |
++--------------+-------------+
+| 3            | Collective  |
++--------------+-------------+
+| 4            | Rudder      |
++--------------+-------------+
+| 5            | Flight Modes|
++--------------+-------------+
+| 6            | Tuning      |
++--------------+-------------+
+| 7            | Aux         |
++--------------+-------------+
+| 8            | Throttle    |
++--------------+-------------+
+
```

Comment:
@ChristopherOlson   Great job on all of this!!!  One thing I think can be misleading is this table.  I think a little more explanation needs to be given on the fact that RC inputs are separated from Servo outputs as far as channel number is concerned.   I think you can say that the software expects from the RC receiver for Aileron, Elevator, Collective, Rudder and flight modes on these channels.  The other inputs channels like throttle can be moved but the default is on channel 8.  (not sure about tuning)   Then you can go into the defaults for servo channels out and explain that they are specified by the ServoX_function parameter.  I think this would help the uninitiated to understand the software and how it works.  Let me know what you think.